src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -70,6 +73,11 @@
     private static final String STACK_TRACE_ON_USER_EXCEPTION_PROPERTY = "GATK_STACKTRACE_ON_USER_EXCEPTION";
 
     /**
+     * The option specifying a main configuration file.
+     */
+    private static final String gatkConfigFileOption = "--gatk-config-file";
```

Comment:
Yeah - it's not in the best place.  `StandardArgumentDefinitions` makes sense to me.